front/components/assistant/AssistantDetails.tsx

```diff
@@ -446,7 +444,7 @@ function DataSourceSelectedNodes({
 
   return (
     <>
-      {dataSourceSelectedNodes.nodes.map((node: ContentNode) => (
+      {dataSourceSelectedNodes.nodes.map((node: LightContentNode) => (
```

Comment:
Given that we have not yet changed `useDataSourceContentNodes` this should still be a `ContentNode`